src/test/java/com/sevendragons/practice/leetcode/medium/courseschedule/Solution.java

```diff
@@ -0,0 +1,71 @@
+package com.sevendragons.practice.leetcode.medium.coursesschedule;
+
+import java.util.HashSet;
+import java.util.Set;
+import java.util.Stack;
+
+public class Solution {
+
+	public static int	NUM_COURSE;
+	public Map<Integer,Set> accessibleNodes = new HashMap<Integer,Set>();
+
+	public boolean canFinish(int numCourses, int[][] prerequisites) {
+		NUM_COURSE = numCourses;
+		
+		// Init graph
+		int[][] graph = new int[numCourses][numCourses];
+
+		for (int i = 0; i < numCourses; i++) {
+			for (int j = 0; j < numCourses; j++) {
+				graph[i][j] = 0;
+			}
+		}
+
+		for (int i = 0; i < prerequisites.length; i++) {
```

Comment:
It seems this outer loop can be simplified using an enhanced for-each: `for (int[] pair : prerequisites) { ... }`